blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_ru.properties

```diff
@@ -0,0 +1,169 @@
+rundetail.header.branch=\u0412\u0435\u0442\u043a\u0430
+rundetail.header.changes.count={0} \u0438\u0437\u043c\u0435\u043d\u0435\u043d\u0438\u0439
+rundetail.header.changes.names=\u0418\u0437\u043c\u0435\u043d\u0435\u043d\u0438\u044f \u043f\u043e {0}
+rundetail.header.changes.none=\u0411\u0435\u0437 \u0438\u0437\u043c\u0435\u043d\u0435\u043d\u0438\u0439
+rundetail.header.commit=\u041a\u043e\u043c\u043c\u0438\u0442
+rundetail.header.tab.artifacts=\u0410\u0440\u0442\u0435\u0444\u0430\u043a\u0442\u044b
+rundetail.header.tab.changes=\u0418\u0437\u043c\u0435\u043d\u0435\u043d\u0438\u044f
+rundetail.header.tab.pipeline=\u041a\u043e\u043d\u0432\u0435\u0439\u0435\u0440
+rundetail.header.tab.tests=\u0422\u0435\u0441\u0442\u044b
+rundetail.input.cancel=\u041f\u0440\u0435\u0440\u0432\u0430\u0442\u044c
+rundetail.pipeline.description=\u041e\u043f\u0438\u0441\u0430\u043d\u0438\u0435
+rundetail.pipeline.logs=\u041b\u043e\u0433\u0438
+rundetail.pipeline.nosteps.message.title=\u041d\u0435\u0442 \u0448\u0430\u0433\u043e\u0432
+rundetail.pipeline.nosteps.message.description=\u042d\u0442\u043e\u0442 \u044d\u0442\u0430\u043f \u043d\u0435 \u0438\u043c\u0435\u0435\u0442 \u0448\u0430\u0433\u043e\u0432
+rundetail.pipeline.waiting.message.title=\u0412 \u043e\u0447\u0435\u0440\u0435\u0434\u0438
+rundetail.pipeline.waiting.message.description=\u041e\u0436\u0438\u0434\u0430\u043d\u0438\u0435 \u0437\u0430\u043f\u0443\u0441\u043a\u0430
+rundetail.pipeline.pending.message.title=\u0417\u0430\u0433\u0440\u0443\u0437\u043a\u0430
+rundetail.pipeline.pending.message.description=
+rundetail.pipeline.steps=\u0428\u0430\u0433\u043e\u0432 {0}
+rundetail.tests.duration=\u041f\u0440\u043e\u0434\u043e\u043b\u0436\u0438\u0442\u0435\u043b\u044c\u043d\u043e\u0441\u0442\u044c {0}
+rundetail.tests.failed=\u041f\u0440\u043e\u0432\u0430\u043b\u0435\u043d\u043e {0}
+rundetail.tests.passed=\u041f\u0440\u043e\u0439\u0434\u0435\u043d\u043e {0}
+rundetail.tests.results.empty.linkhref=https\://jenkins.io/doc/pipeline/tour/tests-and-artifacts/
+rundetail.tests.results.empty.linktext=\u041f\u043e\u0434\u0440\u043e\u0431\u043d\u0435\u0435
+rundetail.tests.results.empty.title=\u0414\u043b\u044f \u0434\u0430\u043d\u043d\u043e\u0433\u043e \u043f\u0440\u043e\u0433\u043e\u043d\u0430 \u043d\u0435 \u043f\u043e\u043b\u0443\u0447\u0435\u043d\u043e \u0442\u0435\u0441\u0442\u043e\u0432
+rundetail.tests.results.error.message=\u041e\u0448\u0438\u0431\u043a\u0430
+rundetail.tests.results.error.output=\u0422\u0440\u0435\u0439\u0441
+rundetail.tests.results.error.stdout=\u0421\u0442\u0430\u043d\u0434\u0430\u0440\u0442\u043d\u044b\u0439 \u043f\u043e\u0442\u043e\u043a \u0432\u044b\u0432\u043e\u0434\u0430
+rundetail.tests.results.error.stderr=\u0421\u0442\u0430\u043d\u0434\u0430\u0440\u0442\u043d\u044b\u0439 \u043f\u043e\u0442\u043e\u043a \u043e\u0448\u0438\u0431\u043e\u043a
```

Comment:
I also suprized that this is two different UI strings. So being unsure about usage context I translated both with full technical term. Your variant looks better though.